src/library/waveform.js

```diff
@@ -0,0 +1,57 @@
+// This is the library for creating the waveform
+class Waveform {
```

Comment:
Can we add more details here? How does it draw/visualize waveform? Any notable constraints/restriction/dependencies?